### ISSUE TITLE
Module parse failed 

### ISSUE THREAD

**Issue by mflmartin** [0rpc/zerorpc-node#88]
Hi, 

After installing zerorpc in node, and using it in an ionic2 app with electron (either in ionic serve / npm start -with electron-) I get this error:

`Module parse failed: D:\Sync\Martin\inProgress\P4Q\_bulk\zeromqtest\zeromqtest\node_modules\zeromq\build\Release\zmq.node Unexpected character '�' (1:2) You may need an appropriate loader to handle this file type. (Source code omitted for this binary file)`

I tried rebuilding the sources, but nothing. 

Any ideas? Thanks,

**Comment by KrustyC** [comment]
Any development on this? I have the same issue

**Comment by salvolm84** [comment]
Same here, electron-quasar app. I think has something to do with webpack that does not know what to do with this *.node file and it is looking for a suitable loader.
I'm pretty stuck, hope someone can help on this

**Comment by salvolm84** [comment]
oh yes, I think this can help https://github.com/webpack-contrib/node-loader
Testing it right now

**Comment by salvolm84** [comment]
Yes that was it, just install the node-loader and update your webpack config accordingly

**Comment by rakakhrl** [comment]
> Yes that was it, just install the node-loader and update your webpack config accordingly

i have the same issue but im using babel
